routes/v1.js

```diff
@@ -1270,9 +1317,13 @@ api.declare({
 
   // Check if task is past deadline
   if (task.deadline.getTime() <= Date.now()) {
-    return res.status(409).json({
-      message: "Task deadline exceeded"
-    });
+    return res.reportError(
+      "ResourceExpired",
```

Comment:
Conflict again, I would say...
As the run we're trying to reclaim still exists.. .it's just not in a state where it can be reclaimed.

I think `410` is reserved for when something is "gone", as in the resource addressed isn't there anymore. We'll rarely have this, as we'll often just return 404, because the resource isn't there :) And we have no record that it ever was...

Maybe one day in artifacts, we'll store the record that the artifact existed after we delete the S3 object...  But we're not doing that now.. (ie. the artifact metadata record could exist until the task expires, but the s3 object could expire sooner -- in some future).